.github/workflows/e2ePerformanceTests.yml

```diff
@@ -0,0 +1,158 @@
+name: E2E Performance Tests
+
+on:
+  workflow_call:
+    inputs:
+      DELTA_REF:
+        description: A branch or commit ref to build the test build on
+        type: string
+        required: true
+      PR_NUMBER:
+        description: The number of a merged PR, if this workflow is being triggered by a PR merge
+        type: string
+        required: true
+
+  workflow_dispatch:
+    inputs:
+      DELTA_REF:
+        description: A branch or commit ref to build the test build on
+        type: string
+        required: true
```

Comment:
Do we need to define PR_NUMBER here too?

Reply:
I don't think so, since the `PR_NUMBER` is only used if the workflow is triggered by the `workflow_call` event, and it's only used to leave comments on the PR